src/utils/generate-type-from-schema.ts

```diff
@@ -0,0 +1,33 @@
+import { compileFromFile } from 'json-schema-to-typescript'
+import fs from 'fs'
+import R from 'ramda'
+import pascalCase from 'pascal-case'
+
+export const generateTypeFromSchema = async (
+  filePath: string
+): Promise<void> => {
+  const baseTypes = await compileFromFile(filePath)
+
+  const schemaString = fs.readFileSync(filePath).toString()
+  const title = R.prop('title', JSON.parse(schemaString))
+
+  if (title === undefined) {
+    throw new Error('Expected title attribute in top-level schema definition.')
+  }
+
+  if (title.toLowerCase() === 'config') {
+    throw new Error(
+      'Title attribute of top-level schema definition must not be named Config or config'
+    )
+  }
+
+  const configInterfaceAsString = `export interface Config extends ${pascalCase(
+    title
+  )} {
+  runtimeEnvironment: string;
+}
+`
+  const exportedTypes = baseTypes.concat(configInterfaceAsString)
+
+  fs.writeFileSync('strongconfig.d.ts', exportedTypes)
```

Comment:
nit: a bit off-brand. npm package is called `@strong-config` and github or is called `strong-config`, so i'd also make this filename `strong-config.d.ts`